docs/index.html

```diff
@@ -23,7 +23,7 @@
 <body>
 <header>
 <h1>The Dafny Programming and Verification Language</h1>
-<!-- VERSION INFO HERE-->Development version
+<!-- VERSION INFO HERE--><center>Development version</center>
```

Comment:
I think you [should](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center) use `text-align` here.

Reply:
Done